app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -318,6 +342,130 @@ ControllerContainer::GetControllers() const {
   return m.list;
 }
 
+template <typename GenericVector>
+static bool loadHandMeshAttribute(tinygltf::Model& model,
+                                  int accessorIndex, int expectedType, int expectedComponentType,
+                                  size_t typeByteSize, int numComponents,
+                                  std::vector<GenericVector>& vector) {
+  auto& accessor = model.accessors[accessorIndex];
+
+  if (accessor.type != expectedType || accessor.componentType != expectedComponentType)
+    return false;
+  if (accessor.bufferView >= model.bufferViews.size())
+    return false;
+  auto& bufferView = model.bufferViews[accessor.bufferView];
+  if (bufferView.buffer >= model.buffers.size())
+    return false;
+  auto& buffer = model.buffers[bufferView.buffer];
+
+  vector.resize(accessor.count);
+  memcpy(vector.data(), buffer.data.data() + bufferView.byteOffset + accessor.byteOffset,
+         accessor.count * typeByteSize * numComponents);
+
+  return true;
+}
+
+bool ControllerContainer::LoadHandMeshFromAssets(Controller& aController) {
+  auto& handMesh = aController.handMesh;
+
+  tinygltf::TinyGLTF modelLoader;
+  tinygltf::Model model;
+  std::string err;
+  std::string warn;
+  bool ok = modelLoader.LoadBinaryFromFile(&model, &err, &warn,
+                                           aController.leftHanded ? "hand-model-left.glb" : "hand-model-right.glb");
+  if (!ok) {
```

Comment:
`ok` is not really needed so I think we can just do `if (!modelLoader....) { VRB_ERROR... ; return false }`

Reply:
Ok